os/sys/autostart.h

```diff
@@ -52,7 +54,18 @@ struct process * const autostart_processes[] = {__VA_ARGS__, NULL}
 
 extern struct process * const autostart_processes[];
 
-void autostart_start(struct process * const processes[]);
-void autostart_exit(struct process * const processes[]);
+static inline void autostart_start(struct process * const processes[])
```

Comment:
Please add a new line before `autostart_start`. Same for `autostart_exit`.

Reply:
Fixed.